__tests__/app.test.js

```diff
@@ -29,6 +29,40 @@ describe("/api/topics", () => {
     });
   });
 });
+describe("/api", () => {
+  describe("GET request", () => {
+    test("responds with json representation of all the available endpoints of the api", () => {
+      return request(app)
+        .get("/api")
+        .expect(200)
+        .then(({ body: { endpoints } }) => {
+          expect(endpoints).toEqual(endpoints);
+        });
+    });
+  });
+});
```

Comment:
This test is always passing, if you balance `endpoints` on one side of the expect, and also `endpoints` in the toEqual then you are testing whether `endpoints === endpoints` which is always true
You could be requiring in the endpoints.json and using that to test that the endpoints in your response matches the endpoints in your file